ProjectSettings/TagManager.asset

```diff
@@ -3,7 +3,8 @@
 --- !u!78 &1
 TagManager:
   serializedVersion: 2
-  tags: []
+  tags:
+  - IgnoreCollider
```

Comment:
Could we use an existing layer for this? Perhaps IgnoreRaycasts?

Reply:
Yeah the tag is unused as i used a layer instead, will remove